vsm

```diff
@@ -22,6 +23,34 @@ LOGIC_REPLACE = {'||': 'or',
 
 KEYWORDS = ['condition', 'emit']
 
+start_time_ms = 0
+
+class Log(object):
+    '''
+        Utility class for logging messages
+    '''
+
+    '''
+        Log an informative (non-error) message
+    '''
+    @staticmethod
+    def i(msg, timestamp=False):
```

Comment:
I'm mirroring Android's Log module here. I think the names are fairly conventional otherwise (e[rror], w[arning], i[nformational], d[ebug], v[erbose]) and, like gettext's `_()` function, I think the brevity is justifiable and pretty easy to get familiar with.